web/modules/custom/dbcdk_community/src/Plugin/Block/ProfilesBlock.php

```diff
@@ -121,6 +144,13 @@ public function build() {
         'limit' => $this->pagerLimit,
         'offset' => $this->pageNumber * $this->pagerLimit,
       ];
+      if (!empty($this->searchQuery)) {
+        $filter['where'] = [
+          'username' => [
+            'like' => '%' . $this->searchQuery . '%',
+          ],
+        ];
+      }
       $profiles = $this->profileApi->profileFind(json_encode($filter));
```

Comment:
The client will return `NULL` for empty results. This can lead to unelegant type handling later on.

The smartest way to handle this at the moment is to cast the results to be arrays. `NULL` results will be converted to an empty array.